right/src/postponer.h

```diff
@@ -0,0 +1,57 @@
+#ifndef SRC_POSTPONER_H_
+#define SRC_POSTPONER_H_
+
+/**
+ * Postponer function:
+ *
+ * Postponer is activated by `PostponeNCycles(n)` call, which makes the Postponer
+ * active for next at least n cycles. This means that for at least next n update
+ * cycles, key activations will be stored into Postponer's queue rather than
+ * executed by usb_report_updater.
+ *
+ * The `n` from `PostponeNCycles(n)` is stored into `cycles_until_activation`,
+ * and is decremented every update cycle. Once `cycles_until_activation` reaches
+ * zero, Postponer starts replaying enqueued events. After replaying an event, the
+ * value is reset to POSTPONER_MIN_CYCLES_PER_ACTIVATION.
+ *
+ * Postponer becomes inactive once cycles_until_activation is zero and event queue
+ * is empty.
+ */
+
+// Includes:
+
+    #include "key_states.h"
+
+// Macros:
+
+    #define POSTPONER_BUFFER_SIZE 32
+    #define POSTPONER_BUFFER_MAX_FILL (POSTPONER_BUFFER_SIZE-5)
```

Comment:
Why `-5`? Extracting -5 to a properly named macro may be the best way to explain it.

Reply:
Quite arbitrary number. Added some explanation...